### PR TITLE
docs(examples): add workshop 2022-11-14 examples

### PR DESCRIPTION
**Pull request by RafaelAPB**
Simple examples using Cacti tailored for the workshop.

Thanks to @AndreAugusto11  and Monica Gomez <monicasrgomez@gmail.com> @msrgomez for the amazing work.  

Works with version 1.0.0 of cactus-tooling.  

Depends on #2213 
Depends on #2223

### CODE REVIEW

File: yarn.lock
Comment (petermetz) [inline yarn.lock:2817]: @RafaelAPB Let's wait for the fix to the substrate test ledger (hopefully in v1.1.3) and that way we can avoid adding an old dependency to the lock file like the v1.0.0 test tooling package (could lead to interesting build failures and confusion).

Please:
1. Create a bug in the issue tracker for the substrate test ledger problem we discovered the morning of the workshop
2. Mark this PR dependent on that.
3. That way when that issue gets fixed we'll know that this one is ready for prime time (at which time I'll ask you to bump the test-tooling package dependency version to the one with the fix)

File: examples/cactus-workshop-examples/package.json
Comment (petermetz) [inline examples/cactus-workshop-examples/package.json:3]: @RafaelAPB Please when the time comes bump this version to whatever the next release will be at the time.

**Review by petermetz — CHANGES_REQUESTED** [review]

**Comment by takeutak** [comment]
@RafaelAPB Thanks for your contribution and hard work to develop a workshop example!
As Peter suggested, the directory name (`cactus-workshop-example`) should be changed to some identified names such as `cacti-workshop-examples-2022-11-14` because there will be another workshop example in the future.

File: yarn.lock
Comment (RafaelAPB) [reply yarn.lock:2817]: Done. Could we instead accept the PR with test-tooling version 1.0.0 (one that passes), so that we can use the code on the PT workshop, next week? @petermetz

File: examples/cactus-workshop-examples/package.json
Comment (RafaelAPB) [reply examples/cactus-workshop-examples/package.json]: Can you provide an example?

**Comment by RafaelAPB** [comment]
@petermetz and @takeutak, the updates you recommended have been incorporated

File: examples/cactus-workshop-examples/package.json
Comment (petermetz) [reply examples/cactus-workshop-examples/package.json]: > Can you provide an example? The watch script is already included

@RafaelAPB Nah, my bad. I mixed it up with the open API watch. Sorry!

**Review by petermetz — CHANGES_REQUESTED** [review]
> @RafaelAPB Both the description and the package name (and directory name) should be pinned to the actual workshop instance (2022-11-14) so that we can add more of these packages later

1. Please fix the above request from earlier and
2. Squash into a single commit
3. Update the tsconfig.json to match the conventions of the other packages if possible (if not possible for whatever reason, then let's discuss the details)
4. Mark the issue dependent with the correct syntax both in the PR description and the commit message.
5. Set the PR title to match the final remaining (one) commit's subject line.
6. Make sure that the commit subject line is also identifying the workshop instance.
7. Make sure that the README title is also identifying the workshop instance.

**Comment by takeutak** [comment]
@RafaelAPB 
> @petermetz and @takeutak, the updates you recommended have been incorporated
Thanks.   It is no problem for me, but could you revise this pull-request according to Peter's request?  After that I will approve it.

**Comment by RafaelAPB** [comment]
@

> > @RafaelAPB Both the description and the package name (and directory name) should be pinned to the actual workshop instance (2022-11-14) so that we can add more of these packages later
> 
> 1. Please fix the above request from earlier and
> 2. Squash into a single commit
> 3. Update the tsconfig.json to match the conventions of the other packages if possible (if not possible for whatever reason, then let's discuss the details)
> 4. Mark the issue dependent with the correct syntax both in the PR description and the commit message.
> 5. Set the PR title to match the final remaining (one) commit's subject line.
> 6. Make sure that the commit subject line is also identifying the workshop instance.
> 7. Make sure that the README title is also identifying the workshop instance.

@petermetz updated. Not sure about the changes required for steps 3 and 4.

**Review by petermetz — CHANGES_REQUESTED** [review]
@RafaelAPB If you compare your tsconfig.json (packages/cactus-plugin-ledger-connector-besu/tsconfig.json) to for example the besu connector one, you'll catch a couple of differences:
1. The include paths are universal (more or less) whereas yours are package specific
2. The build cache you specified puts the file in a sub-folder while the others aren't (by convention to have a flat file structure for the build-cache files)

For point 4 see https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

And also note that you can mark dependencies the same way but instead of saying "fixes #????" you can say "depends on #????" and then the robots will notice that and block the PR from getting merged accidentally before the parent PR (the one this depends on) has been merged.

The commit message is still not consistent with the PR description, it's fine to add *more* information to the commit message compared to the PR description but not the other way around.

**Comment by RafaelAPB** [comment]
@petermetz updated!

**Review by jagpreetsinghsasan — APPROVED** [review]
LGTM, its nice to have such simple examples as well, for getting started.

**Review by takeutak — APPROVED** [review]

**Comment by takeutak** [comment]
LGTM, but this pull-request cannot be merged without resolving this change-request from Peter due to GitHub system.

@petermetz If you have some improvement comments, please feel free to share them.

**Review by petermetz — CHANGES_REQUESTED** [review]
LGTM but will mark it with a change request so that we know that it has to be updated after v1.1.3 comes out (which is dropping soon)

**Review by petermetz — CHANGES_REQUESTED** [review]
@RafaelAPB Please update to v1.1.3 and then if you run `yarn` it should disappear the unnecessary diff from the lock file.

**Review by petermetz — CHANGES_REQUESTED** [review]
@RafaelAPB You don't appear to have ran `yarn` to make the lock file changes necessary.

**Comment by RafaelAPB** [comment]
> @RafaelAPB You don't appear to have ran `yarn` to make the lock file changes necessary.

should be good to go now! Thanks for the patience

**Review by petermetz — APPROVED** [review]
@RafaelAPB No worries and thanks for the updates, LGTM.

**Pull request merged**
